features/storage/FEATURE_STORAGE/TESTS/cfstore/add_del/add_del.cpp

```diff
@@ -136,10 +136,10 @@ control_t cfstore_add_del_test_01_end(const size_t call_count)
 
 
 static cfstore_kv_data_t cfstore_add_del_test_08_data[] = {
-        CFSTORE_INIT_1_TABLE_HEAD,
-        CFSTORE_INIT_1_TABLE_MID_NODE,
-        CFSTORE_INIT_1_TABLE_TAIL,
-        { NULL, NULL},
+    CFSTORE_INIT_1_TABLE_HEAD,
+    CFSTORE_INIT_1_TABLE_MID_NODE,
+    CFSTORE_INIT_1_TABLE_TAIL,
+    { NULL, NULL},
```

Comment:
Not sure what you require here. Astyle left this untouched. Changing to {NULL, NULL}. Hope this is as intended.